developers/compute-to-data/compute-workflow.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Compute Workflow
+section: developers
+description: Understanding the Compute-to-Data (C2D) Workflow
+---
+
+🚀  Now that we've introduced the key actors and provided an overview of the process, it's time to delve into the nitty-gritty of the compute workflow. We'll dissect each step, examining the inner workings of Compute-to-Data (C2D). From data selection to secure computations, we'll leave no stone unturned in this exploration. 
+
+For visual clarity, here's an image of the workflow in action! 🖼️✨
+
+<figure><img src="../../.gitbook/assets/c2d/c2d_detailed_flow.png" alt=""><figcaption><p>Compute detailed flow diagram</p></figcaption></figure>
+
+Below, we'll outline each step in detail 📝
+
+## Starting a C2D Job
+1. The consumer initiates a compute-to-data job by selecting the desired data asset and algorithm. 
+2. The provider checks the orders on the blockchain.
+3. If the orders are valid, the provider can commence the compute flow.
+4. The provider informs the consumer of the job number's successful creation.
+5. With the job ID and confirmation of the orders, the consumer starts the job by calling the operator service.
+6. The operator service communicates with the operator engine to initiate a new job.
+
+## Creating the K8 Cluster and Allocating Job Volumes
+7. As a new job begins, volumes are created on the Kubernetes cluster, a task handled by the operator engine.
+8. The cluster creates and allocates volumes for the job using the job volumes
+9. The volumes are created and allocated to the pod
+10. After volume creation and allocation, the operator engine initiates the `run script` on the pod configuration.
```

Comment:
I'd suggest the following: 10. After the volume creation and allocation, the Operator Engine calls the POD Configuration script.